### PR TITLE
bzr-builder 0.7.3 (new formula)

### PR DESCRIPTION
**Pull request by fmccann**

### CODE REVIEW

File: Library/Formula/bzr-builder.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    cd "repo" do
+      system "bzr", "init", "trunk"
+
+      cd "trunk" do
+        touch "readme.txt"
+        system "bzr", "add"
+        system "bzr", "commit", "-m", "initial import"
+      end
+
+      open("my.recipe", "w") { |f| f.puts "# bzr-builder format 0.3 deb-version 1.0+{revno}-{revno:packaging}\ntrunk" }
```
Comment (tdsmith) [inline Library/Formula/bzr-builder.rb]: We like to use Pathname methods for this:
`(testpath/"my.recipe").write "# bzr-builder format 0.3 deb-version 1.0+{revno}-{revno:packaging}\ntrunk"`

File: Library/Formula/bzr-builder.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      system "bzr", "build", "my.recipe", "branch"
+
+      cd "branch" do
+        assert_match("bzr-builder.manifest", shell_output("ls -la"))
+        assert_match("readme.txt", shell_output("ls -la"))
+      end
```
Comment (tdsmith) [inline Library/Formula/bzr-builder.rb]: Similarly:
```
assert (testpath/"branch/bzr-builder.manifest").exist?
assert (testpath/"branch/readme.txt").exist?
```

**Comment by tdsmith** [comment]
Thanks for the PR! lgtm except for test nits.

File: Library/Formula/bzr-builder.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    cd "repo" do
+      system "bzr", "init", "trunk"
+
+      cd "trunk" do
+        touch "readme.txt"
+        system "bzr", "add"
+        system "bzr", "commit", "-m", "initial import"
+      end
+
+      (testpath/"repo/my.recipe").write "# bzr-builder format 0.3 deb-version 1.0+{revno}-{revno:packaging}\ntrunk"
```
Comment (DomT4) [inline Library/Formula/bzr-builder.rb]: It'd probably be nice to wrap this in `<<-EOS.undent` so it doesn't run past the 80 character recommendation with the excitement of a Star Wars fan in the December just passed.

**Comment by tdsmith** [comment]
Thanks for your pull request! :tada: Homebrew thrives because of contributions like this one.

**Comment by apjanke** [comment]
Closed manually since the "Closes #49286" commit comment looks like it got lost while squashing commits.

**Pull request closed without merging**